### ISSUE TITLE
customFields is always null at reach5.getProfile()

### ISSUE THREAD

**Issue by dirceudn** [ReachFive/identity-android-sdk#48]
When I call the update profile like this:

```
 val profile = Profile(gender = gender, name = name, familyName = lastName,
                locale = address, birthdate = date, customFields = fields)
 reach5.updateProfile(token!!, profile = profile, success = {
            Timber.tag(TAG).d("Profile $it")
            handleSuccess()
        }, failure = { error ->
            handleError(error)
        })
```
The updateProfile method is good:
```

addresses = null
authTypes = {ArrayList@20079}  size = 1
bio = null
birthdate = "2017-12-20"
company = null
consents = null
createdAt = "2019-08-06T13:48:46.418Z"
customFields = {LinkedTreeMap@20082}  size = 2
 0 = {LinkedTreeMap$Node@20100} "mobile_number1" -> 
 1 = {LinkedTreeMap$Node@20101} "email_address1" -> "sample@gmail.com"
email = null
emailVerified = {Boolean@20007} false
emails = {Emails@20083} "Emails(verified=[], unverified=[])"
externalId = null
familyName = "Teixeira"
gender = "male"
givenName = null
liteOnly = {Boolean@20007} false
locale = "gdhdh"
loginSummary = {LoginSummary@20087} "LoginSummary(firstLogin=1565099326659, lastLogin=1565705232464, total=41, origins=[], devices=[android], lastProvider=phone_number_password)"
middleName = null
name = "Teixeira"
nickname = null
phoneNumber = "+33999999999"
phoneNumberVerified = {Boolean@20015} true
picture = null
profileURL = null
signedUid = null
tosAcceptedAt = null
uid = "AWxnMHvUpj2ZiTp0fD7p"
updatedAt = "2019-08-13T14:12:50.571Z"
```
But the getProfile is not working well and the customFields is always null:

```
addresses = null
authTypes = {ArrayList@20169}  size = 1
bio = null
birthdate = "2017-12-20"
company = null
consents = null
createdAt = "2019-08-06T13:48:46.418Z"
customFields = null
email = null
emailVerified = {Boolean@20007} false
emails = {Emails@20172} "Emails(verified=[], unverified=[])"
externalId = null
familyName = "Teixeira"
gender = "male"
givenName = null
liteOnly = {Boolean@20007} false
locale = "gdhdh"
loginSummary = {LoginSummary@20176} "LoginSummary(firstLogin=1565099326659, lastLogin=1565705818537, total=42, origins=[], devices=[android], lastProvider=phone_number_password)"
middleName = null
name = "Teixeira"
nickname = null
phoneNumber = "+3399999999"
phoneNumberVerified = {Boolean@20015} true
picture = null
profileURL = null
signedUid = null
tosAcceptedAt = null
uid = "AWxnMHvUpj2ZiTp0fD7p"
updatedAt = "2019-08-13T14:16:58.537Z"
username = null
```

**Comment by roxanemace** [comment]
Hello, I've just tested it on my side and I get correctly the custom fields. Are you sure that the keys of your `fields` object match the custom fields set on your ReachFive console?

**Comment by roxanemace** [comment]
Hello @dirceudn,
Unfortunately, this is a known bug. You cannot get the values of a custom field which key contains a number. Hopefully, this should be fixed in our next iteration. Meanwhile, I suggest you rename your keys `mobile_number_one` and `email_address_one`.

**Comment by dirceudn** [comment]
Ok @roxanemace Thanks =)

**Comment by roxanemace** [comment]
Hi @dirceudn,

The problem was finally fixed with the [v5.4.2](https://github.com/ReachFive/identity-android-sdk/releases/tag/v5.4.2) release. 😄

**Comment by dirceudn** [comment]
Hi @roxanemace yes the customFields are ok. But now I have the phoneNumberVerified and EmailVerified null. Before both (phoneNumberVerified and EmailVerified) were  false by default.

Thanks,

**Comment by roxanemace** [comment]
Looking right now!

**Comment by dirceudn** [comment]
Thanks

**Comment by roxanemace** [comment]
Hi @dirceudn,
The problem should be resolved with the [v5.4.3](https://github.com/ReachFive/identity-android-sdk/releases/tag/v5.4.3) release.

**Comment by dirceudn** [comment]
Ok. Thanks
